esp/esp/survey/templatetags/survey.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import division
```

Comment:
Is this necessary? If you're worried about getting a float (or not) from your division, you can just divide by a float (or not) (e.g. 3/2 = 1, 3/2. = 1.5).